.settings/module_db.json

```diff
@@ -1,7 +1,7 @@
 {
-  "build_number": 7,
-  "build_date": "20-03-2024 12:51:58",
-  "git_revision_number": "3be6f3847150d4bb8debee2451522b0b19fa205f",
+  "build_number": 8,
+  "build_date": "11-04-2024 13:03:07",
```

Comment:
Please pull the latest version of the dev branch to update your build_nummer, modules and necessary packages. There should be no dependency conflicts.

Reply:
Should it not have a new build_number as the new MIS modules and dependencies were added to the file?
I might misunderstand what the build number date and git revision number represent.

Reply:
> Please pull the latest version of the dev branch to update your build_nummer, modules and necessary packages. There should be no dependency conflicts.

It does seem like IBM eagle is supported in main but not on dev.
Because I copied the module_db from dev now, it would remove IBM eagle support, is that intentionally dropped in dev?